test/distributed/_tensor/test_math_ops.py

```diff
@@ -187,6 +187,12 @@ def _replicate_fn(name, module, device_mesh):
                 device_mesh, placements=[Replicate()]
             )
 
+            from torch.distributed._tensor.placement_types import TensorMeta
+
+            dtensor_meta = y_dist._spec.tensor_meta
+            assert isinstance(dtensor_meta, TensorMeta)
```

Comment:
hmmm instead of importing the TensorMeta, if you want to just check the tensor meta shape, you can simply do `y_dist._spec.shape`

Reply:
Will apply change in layer norm backward PR